server/models/views/target-area.js

```diff
@@ -87,4 +87,8 @@ class ViewModel {
   }
 }
 
+function messageValidator (message) {
+  const strippedMessage = message.replace(/(\r?\n)+/g, '\n')
```

Comment:
Do we have any incidences of warnings containing '\r\n'.

My understanding is that this is not a dos/unix type issue, it's more to do with a convention employed in the alerts and warnings to include the string '\n' where a new line should (?could) be.

Reply:
From meeting I had with Ray/Kim the \n is just generated every time the user hits the return key whilst typing. Discussed with Max and we weren't sure whether there was therefore a chance this could come from either Windows of Unix machine so altered this to cover all bases.